jsonschema-core.xml

```diff
@@ -1593,6 +1597,60 @@
                         </t>
                     </section>
 
+                    <section title="unevaluatedItems" anchor="unevaluatedItems">
+                        <t>
+                            The value of "unevaluatedItems" MUST be a valid JSON Schema.
+                        </t>
+                        <t>
+                            The behavior of this keyword depends on the annotation results of
+                            adjacent keywords that apply to the instance location being validated.
+                            Specifically, the annotations from "items" and  "additionalItems",
+                            which can come from those keywords when they are adjacent to the
+                            "unevaluatedItems" keyword.  Those two annotations, as well as
+                            "unevaluatedItems", can also result from any and all adjacent
+                            <xref target="in-place">in-place applicator</xref> keywords.
+                            This includes but is not limited to the in-place applicators
+                            defined in this document.
+                        </t>
+                        <t>
+                            If an "items" annotation is present, and its annotation result
+                            is a number, and no "additionalItems" or "unevaluatedItems"
```

Comment:
I'm being dumb. Can you explain / give an example where an annotation result is a number please?

Reply:
It's explained in the section on "items", and we're talking about the "items" annotation.  This is also identical to the language used in describing `additionalItems`, I literally copied and pasted that part IIRC.